linetools/spectra/xspectrum1d.py

```diff
@@ -143,9 +143,11 @@ def __init__(self, wave, flux, sig=None, co=None, units=None, select=0,
         sig : ndarray, optional
         units : dict, optional
           Dict containing the units of wavelength, flux
-          Required keys are 'wave' and 'flux'
+          Required keys are 'wave' and 'flux li
         select : int, optional
           Selected Spectrum
+        mask_edges : bool, optional
+          Mask the edges of each spectrum where sig <= 0.
```

Comment:
It does not.  Hence the named 'edges'.  
I'm adding in a mask_all which will do all sig<=0